lib/nationbuilder/client.rb

```diff
@@ -68,6 +68,26 @@ def response
     Thread.current[:nationbuilder_rb_response]
   end
 
+  def next?
+    @next ? true : false
```

Comment:
There's no need for the `? true : false` bit. If `@next` not set, that's considered a falsey value. The same applies to `prev?`